src/dsp_tools/commands/xmlupload/xmlupload.py

```diff
@@ -260,7 +259,7 @@ def _upload_one_resource(
     iri = None
     try:
         serialised_resource = create_resource_with_values(
-            resource=transformed_resource, bitstream_information=media_info, lookup=iri_lookup
+            resource=transformed_resource, bitstream_information=media_info, lookups=iri_lookup
```

Comment:
I still feel uneasy with the naming here. You have a variable `iri_lookup`, but the class name is `IRILookups`. But the content of the class are several fields, so the name doesn't fit the content.